web/ui/react-app/src/pages/targets/Filter.tsx

```diff
@@ -7,6 +7,10 @@ export interface FilterData {
   showUnhealthy: boolean;
 }
 
+interface Blah<T> {
+  fn: (t: T) => T;
+}
+
 export interface FilterProps {
   filter: FilterData;
   setFilter: React.Dispatch<React.SetStateAction<FilterData>>;
```

Comment:
You can omit React namespace here and use named imports instead

Reply:
done! Thanks